### PR TITLE
Add new repo guide

### PR DESCRIPTION
**Pull request by osanseviero**
This guide combines some course and documentation content. The idea is that if someone asks "How do I upload my models to the Hub" or if we ask someone in Twitter or in a GitHub issue if they are interested in uploading their models to the Hub, we directly point them to this guide.

The guide is focused on quickly uploading things to the Hub.

### CODE REVIEW

File: docs/hub/adding-a-model.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+![/docs/assets/hub/files.png](/docs/assets/hub/files.png)
+
+
+## Uploading your files
+
+If you've used Git before, this will be very easy since Git is used to manage files in the Hub.
+
+There is only one key difference if you have large files (over 10MB). These files are usually tracked with **git-lfs** (which stands for Git Large File Storage). 
+
+1. Please make sure you have both **git** and **git-lfs** installed on your system.
```
Comment (LysandreJik) [inline docs/hub/adding-a-model.md:51]: A link to git/git-lfs installation instructions would be welcome

File: docs/hub/adding-a-model.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+```
+git clone https://huggingface.co/<your-username>/<your-model-id>
+```
+
+The directory should contain the `README.md` file created in the previous section.
+
+
+4. Add your files to the repository
+
+Now it's the time 🔥. You can add any files you want to the repository.
```
Comment (LysandreJik) [inline docs/hub/adding-a-model.md:70]:
```suggestion
Now's the time 🔥. You can add any files you want to the repository. 
```

**Review by LysandreJik — APPROVED** [review]
Great, thanks @osanseviero

File: docs/hub/adding-a-model.md (hunk 1 above)
Comment (osanseviero) [reply docs/hub/adding-a-model.md:51]: Great idea!

**Review by julien-c — APPROVED** [review]
looks good!

**Pull request merged**